packages/manager/modules/pci/src/projects/project/instances/instance/detach-private-network/translations/Messages_de_DE.json

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
You have to just add translation for French. Other translation will be done by different team. Please remove all the translation except french one.

Reply:
done